sdk/highlight-node/tsup.config.ts

```diff
@@ -0,0 +1,18 @@
+import { defineConfig } from 'tsup'
+
+export default defineConfig({
+	entry: ['src/index.ts'],
+	format: ['cjs', 'esm'],
+	dts: true,
+	sourcemap: true,
+	noExternal: [
```

Comment:
Bundle all of `@opentelemetry/*`

Reply:
interesting :D this might allow getting https://github.com/open-telemetry/opentelemetry-js/commit/5615b1c1857606e2ba5e591c924eb45448dbcb61 sooner by allowing us to change the version to our fork

Reply:
good to know that's what `noExternal` does!